public/apps/account/log-out-button.tsx

```diff
@@ -16,43 +16,46 @@
 import React from 'react';
 import { EuiButtonEmpty } from '@elastic/eui';
 import { HttpStart } from 'opensearch-dashboards/public';
-import { logout, samlLogout } from './utils';
+import { externalLogout, logout, openidLogout, samlLogout } from './utils';
```

Comment:
Please remove `openidLogout` from import if is not referenced in this file.

Reply:
Addressed.